signal_timeout_block.py

```diff
@@ -38,18 +36,22 @@ def __init__(self):
         self._jobs = defaultdict(dict)
         self._jobs_locks = defaultdict(Lock)
         self._repeatable_jobs = defaultdict(dict)
+        self._single_jobs = defaultdict(dict)
         self._persistence_scheduled = Event()
 
     def persisted_values(self):
         """Use persistence mixin"""
-        return ["_repeatable_jobs"]
+        return ["_repeatable_jobs", "_single_jobs"]
 
     def start(self):
         super().start()
         # Schedule persisted jobs
         for key, intervals in self._repeatable_jobs.items():
             for interval, job in intervals.items():
                 self._schedule_timeout_job(job, key, interval, True)
+        for key, intervals in self._single_jobs.items():
+            for interval, job in intervals.items():
+                self._schedule_timeout_job(job, key, interval, False)
```

Comment:
These `job` objects will be dictionaries, not signals here. So we probably need to convert here. Otherwise the `signal.timeout` line below is going to cause issues like this
```
[2018-06-27T16:47:56.803870Z] NIO [ERROR] [MonitorSensorExts.NIOThread] Executing: _timeout_job
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/nio/util/threading/thread.py", line 33, in run
    self.nio_result = self._target(*self._args, **self._kwargs)
  File "/nio/project/blocks/signal_timeout/signal_timeout_block.py", line 99, in _timeout_job
    signal.timeout = interval
AttributeError: 'dict' object has no attribute 'timeout'
```